wellregistry/wellregistry/env.py

```diff
@@ -0,0 +1,21 @@
+"""Environment values convenience class."""
+import os
+
+
+class Environment:
```

Comment:
I'm unsure if it's really necessary to define all these configuration parameters in their own class. You can gain the same effect by having defining them as constants in this module.

Reply:
I would agree. For the Flask applications, we have put these directly in the config.py (same purpose as settings.py).